modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextModeAndSettings.vue

```diff
@@ -150,8 +149,14 @@ export default {
           return 'Submit';
         }
       }
+    },
+    hasBeenPublishedThisPageload() {
+      return (this.context.lastPublishedAt > this.mountedAt) || ((this.context.submitted && this.context.submitted.at) > this.mountedAt);
```

Comment:
Detect this based on the updated context object we get when a document is changed via any modal.